### PR TITLE
Community view: truncate community description to 2 lines

### PR DESCRIPTION
**Pull request by Pineirin**
Truncate request details contributors to 2 lines
closes: https://github.com/inveniosoftware/invenio-communities/issues/401
closes: https://github.com/inveniosoftware/invenio-app-rdm/issues/1280
Screenshot from 2022-02-18 13-15-30
Screenshot from 2022-03-01 17-06-02
Screenshot from 2022-03-01 17-06-10

### CODE REVIEW

**Review by ntarocco — APPROVED** [review]

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/communities.js [hunk 1]
```diff
@@ -37,7 +37,9 @@ function ResultsGridItemTemplate({ result, index }) {
         <Card.Header>{result.metadata.title}</Card.Header>
         <Card.Description>
           <div
-            dangerouslySetInnerHTML={{ __html: result.metadata.description }}
+            dangerouslySetInnerHTML={{
+              __html: _truncate(result.metadata.description, { length: 225 }),
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/communities.js:41]: how did you come up with this number?

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/components/Truncate.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+/**
+ * Applying an ellipsis to multiline text
+ * https://stackoverflow.com/questions/33058004/applying-an-ellipsis-to-multiline-text
+ */
+export class Truncate extends Component {
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/components/Truncate.js:8]: how come this code is copied in two repositories?

**Review by ntarocco — COMMENTED** [review]
Can we review the changes together?

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/views/item.overrides [hunk 3]
```diff
@@ -16,9 +16,17 @@
...
+
 .ui.items > .item .meta * {
     margin-right: 0.1em;
-}
\ No newline at end of file
+}
+
+#request-details-contributors{
+  .item{
+    margin: 0 1em 0 0;
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/views/item.overrides:30]: why is this change needed?

Comment (Pineirin) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/views/item.overrides:26]: Addresed via internal conversation.

**Review by jrcastro2 — APPROVED** [review]

**Review by kpsherva — APPROVED** [review]
